Tests/PostgresNIOTests/New/Messages/RowDescriptionTests.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 02.02.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```